starter_code/src/movies.js

```diff
@@ -1,20 +1,87 @@
 /* eslint no-restricted-globals: 'off' */
 // Turn duration of the movies from hours to minutes 
+let turnHoursToMinutes = movies.map(movie => {
```

Comment:
This function is causing an error on my machine. Did you run this function successfully?

Reply:
It was running before, but now I tested it again and I'm seeing a reference error on movies. Not quite sure why, but isn't the movies.js suppose to be taking the data from data.js? So why is giving a reference error on movies.map?